### PR TITLE
Allow general jpyt dungeon search when no leader specified

### LINKED ISSUE #1845: Change a couple labels in `otherinfo`
`XP to Max` to `^expcurve` so that we're telling people what the command is called

`JP Added` to `Added to DB` because it's lying for the NA-only cards

The file to do this in is padinfo/views/otherinfo

### LINKED ISSUE #1852: if no monster is specified in jpyt just put the dungeon, plus パズドラ
パズドラ is pazudora

### LINKED ISSUE #1855: In `^ls` get rid of the backticks around the error messages
This is in the padinfo cog, in padinfo.py

### PR DESCRIPTION
**Pull request by kary5678**
Resolves #1852

Update: also resolves #1845, resolves #1855

### CODE REVIEW

File: padinfo/padinfo.py [hunk 1]
```diff
@@ -1512,30 +1512,26 @@ async def jpdgname(self, ctx, *, search_text):
...
-        dg_qs = await QuerySettings.extract_raw(ctx.author, self.bot, dg_text)
+        if '/' not in search_text:
+            dg_text = search_text
+            monster = None
 
-        monster = await dbcog.find_monster(mon_text, ctx.author.id)
-        if monster is None:
-            return await ctx.send(f"No monster found. This command looks for `/` or `,` as a delimiter, "
-                                  f"maybe try again?")
+        elif '/' in search_text:
```
Comment (RheingoldRiver) [inline padinfo/padinfo.py:1522]: oops I *did* have a bug before where `,` wasn't a valid delimiter, but we *should* support splitting by `,` still, it's easier for people on mobile to type

**Review by RheingoldRiver — COMMENTED** [review]
Looks great! I can only find the one small issue tbh and it's not even an issue really

File: padinfo/padinfo.py [hunk 2]
```diff
@@ -1512,30 +1512,34 @@ async def jpdgname(self, ctx, *, search_text):
 
     @commands.command(aliases=["jydl", "jpyt"], usage="<dungeon_name> / <monster_name>")
     async def jpyoutube(self, ctx, *, search_text):
-        """Attempt to link to a YouTube search of a leader in a dungeon"""
+        """Link to a YouTube search of a dungeon, with an option to specify leader"""
         dbcog = await self.get_dbcog()
         db: "DBCogDatabase" = dbcog.database.database
+
         if '/' in search_text:
```
Comment (RheingoldRiver) [inline padinfo/padinfo.py:1519]: You could try something like:

If '/' in search_text or ',' in search_text:

and then try separating by one or the other

**Comment by kary5678** [comment]
Also took the time to resolve #1845 and #1855 as well

**Pull request merged**